oidc-client-app-samples/android-client-app-sample/app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Missing license header.

Reply:
Fixed in PR: https://github.com/wso2/samples-is/pull/179